app/src/main/java/com/cmput301f20t21/bookfriends/ui/library/owned/OwnedListFragment.java

```diff
@@ -194,8 +194,8 @@ private void openDetailActivity(Book book){
     private Class<? extends BaseDetailActivity> getDetailClass(BOOK_STATUS bookStatus) {
         switch (bookStatus) {
             case ACCEPTED:
-                return AcceptedOwnedDetailActivity.class;
             case BORROWED:
+                return AcceptedOwnedDetailActivity.class;
```

Comment:
this looks like it will break the owner handover functionality to borrower, you should instead create a new detail activity called `BorrowedOwnedDetailActivity` and write all the codes in that activity

and leave both `AcceptedOwnedDetailViewModel `and `AcceptedOwnedDetailActivity` untouched

Reply:
agreed